ansible/ext-inventory.yml

```diff
@@ -0,0 +1,65 @@
+# This playbook updates data at ooni-sysadmin.git/ext/
+---
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - name: get git-root
+      command: git rev-parse --show-toplevel
+      register: extroot
+      changed_when: false
+
+    - set_fact:
+        extroot: '{{ extroot.stdout }}/ext'
+        socks5: '{{ 32767 | random(1024) }}'
+        jumphost: '{{ namecheap_jumphosts | random }}'
+    - name: ssh jumphost
+      command: ssh -D {{ socks5 }} {{ jumphost }} sleep 60
```

Comment:
Also we should ensure that the user can access the ssh server, since the ssh client configuration file may not include this host or use a different HostName.
Ideally we could use the ansible's  [proxy_command](http://docs.ansible.com/ansible/intro_configuration.html#proxy-command) and [ssh_args](http://docs.ansible.com/ansible/intro_configuration.html#ssh-args) for any specific ssh configuration. This way we will avoid the "hackish" way of [connecting to the jumphost](https://github.com/TheTorProject/ooni-sysadmin/blob/c72f7725b3d65d85c8a094b02d80b7d2f4638054/ansible/ext-inventory.yml#L12-L20] and using a command with [curl instead of the uri module](https://github.com/TheTorProject/ooni-sysadmin/blob/c72f7725b3d65d85c8a094b02d80b7d2f4638054/ansible/ext-inventory.yml#L25)